### PR TITLE
Add fellows page

### PR DESCRIPTION
**Pull request by starbelly**
- Initial version of the fellows gallery
 - All text sans the intro provided by the Fellowship working group
 - Move academic papers under new drop down "resources" per simply
 running out of header room

Not unlike many of our pages we shuffle the entries.  

- [x] I acknowledge my contribution to the website does not assert comparative or superlative differences of one product, project, company or individual over another.

# Screenshots

Please attach screenshots and/or gifs of any visible changes to pages here.

Screen Shot 2020-12-10 at 11 37 55 AM

Screen Shot 2020-12-10 at 11 38 52 AM

Screen Shot 2020-12-10 at 11 39 45 AM

Screen Shot 2020-12-10 at 11 40 05 AM

### CODE REVIEW

File: lib/erlef_web/templates/page/fellows.html.eex [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<div id="fellows" class="container">
+  <div>
+    <h1 data-toc-skip="true" style="margin-left:1rem;">Fellows</h1>
+    <p style="margin-left:1rem;">
+    What is a fellow? Those whom have made extraordinary contributions to our community.
```
Comment (starbelly) [inline lib/erlef_web/templates/page/fellows.html.eex:5]: Need input here. Might to broad or not broad enough.

Comment (ferd) [reply lib/erlef_web/templates/page/fellows.html.eex:5]: This is good enough. There might be some value in explaining how they're chosen, but that could also just be a link to the fellows WG

**Review by ferd — APPROVED** [review]

File: lib/erlef_web/templates/page/fellows.html.eex (hunk 1 above)
Comment (starbelly) [reply lib/erlef_web/templates/page/fellows.html.eex:5]: Yeah, agreed, good enough.

**Comment by benoitc** [comment]
content looks fine for me as well.

**Pull request merged**